R/escape.R

```diff
@@ -214,7 +218,16 @@ sql_quote <- function(x, quote) {
   y
 }
 
-
+#' Helper function to convert raw vector to hex string
+#'
+#' @param x Raw vector
+#' @export
+#' @keywords internal
+#' @examples
+#' sql_raw_to_hex(charToRaw("abc"))
+sql_raw_to_hex <- function(x) {
```

Comment:
Does this need to be exported?

Reply:
Removed this function altogether as too trivial